extension/observer/cfgardenobserver/README.md

```diff
@@ -0,0 +1,53 @@
+| [Code Owners](https://github.com/open-telemetry/opentelemetry-collector-contrib/blob/main/CONTRIBUTING.md#becoming-a-code-owner)    | [@crobert-1](https://www.github.com/crobert-1), [@cemdk](https://www.github.com/cemdk), [@tomasmota](https://www.github.com/tomasmota), [@m1rp](https://www.github.com/m1rp), [@jriguera](https://www.github.com/jriguera) |
+
+[development]: https://github.com/open-telemetry/opentelemetry-collector#development
+<!-- end autogenerated section -->
+
+Cloudfoundry is a platform as a service (PaaS) which has implemented their own container technology. The component in charge of providing a local API in each node (also known as "diego-cell") is named Garden.
+
+The `cfgarden_observer` looks at the current host to discover garden containers.
+
+## Example Config
+
+```yaml
+extensions:
+  cfgarden_observer:
+    # url of the garden socket, default to unix:///var/vcap/data/garden/garden.sock 
+    endpoint: my/path/to/garden.sock
+    # determines how often to look for changes in endpoints.
+    refresh_interval: 30s
+
+receivers:
+  receiver_creator:
+    watch_observers: [cfgarden_observer]
+    receivers:
+      prometheus_simple:
+        rule: type == "container" && name == "myapp"
+        config:
+          metrics_path: /metrics
+          endpoint: '`endpoint`'
+```
+
+### Configuration
+
+| Name             | Type   | Default                                  | Docs                                                   |
+|------------------|--------|------------------------------------------|--------------------------------------------------------|
+| refresh_interval | string | 60s                                      | Determines how often to look for changes in endpoints. |
+| endpoint         | string | unix:///var/vcap/data/garden/garden.sock | The endpoint to connect to the garden API.             |
+
+### Endpoint Variables
+
+Endpoint variables exposed by this observer are as follows.
+
+| Variable  | Description                                                                                |
+|-----------|--------------------------------------------------------------------------------------------|
+| type      | `"container"`                                                                              |
```

Comment:
The description here isn't immediately clear, can you clarify? Is this simply saying the value for the  exposed `type` key will always be `container`?

Reply:
Yes, it will always be `container`. We just did the same as the host observer here, do you want me to change both to be clearer or none of them? https://github.com/open-telemetry/opentelemetry-collector-contrib/blob/main/extension/observer/hostobserver/README.md?plain=1#L35

Reply:
Thanks for the reference. My personal preference is to lean on the side of extra information, so I like having the extra description here.